### ISSUE TITLE
Compiler Warnings Standard check tags

### ISSUE THREAD

**Issue by nstander** [stinb/plugins#24]
My understanding was the "Compiler" tag was supposed to allow grabbing groups, like:

-Wall = checks with "Compiler: All"
-Wdefault = checks with "Compiler: Default"
-Wpedantic = checks with "Compiler: Pedantic"

But there are only 81 checks with "Compiler: All" and 403 checks with "Compiler: Default". Shouldn't "Compiler: All" be bigger than "Compiler: Default"? And if -Wall is actually supposed to be checks with "Compiler: All" or "Compiler: Default", why do a lot of the checks with "Compiler: All" also have "Compiler: Default"?

**Comment by nstander** [comment]
While looking at compiler warnings anyway, is this one right? I thought group flags like -Wextra weren't supposed to be checks and even if it's intentional the description doesn't make sense to me.

<img width="791" alt="image" src="https://github.com/user-attachments/assets/8e264d77-c74d-4fdf-986f-a7aeb164f6f0" />

**Comment by RobbyCBennett** [comment]
Yeah, that isn't correct. I would just say "A compiler warning group the following" and list the flags that it controls.

[-Wextra](https://clang.llvm.org/docs/DiagnosticsReference.html#wextra)

**Comment by nstander** [comment]
>  I would just say "A compiler warning group the following" 

But warning groups aren't supposed to be checks at all, are they?

**Comment by RobbyCBennett** [comment]
I don't know. There are a lot of groups, so maybe it does make sense to have warning groups as checks instead of as tags.
